pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -496,17 +496,19 @@ func TestEnoughRequests(t *testing.T) {
 				test.args.ScoringStrategy = defaultScoringStrategy
 			}
 
-			p, err := NewFit(&test.args, nil, plfeature.Features{})
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
```

Comment:
The `ktesting.NewTestContext` function is not used in this location because of the initialization `NewFit` function of the plugin, the `PreFilter`,`Filter` functions do not call `klog` to output logs.

the same below.

Reply:
Using the same way is more uniform, so I'll modify it.